### PR TITLE
BC-73 Feat/automated tests sprint5

### PR DESCRIPTION
**Pull request by jwpradm**
## Description

This PR solves BC-73

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] User stories met
- [ ] Storybook stories
- [ ] Unit tests added
- [ ] Linting passing
- [ ] Unit tests passing
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: test/account_test.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  I.click('Cancel');
+  I.dontSee('Confirm password');
+
+  I.click('Edit account');
+  I.fillField('email', '12345@test.org');
+  I.fillField('confirmationPassword', 'Ax854bZ!$');
+  I.click('Save');
+  I.wait(2);
+  I.see('Email already exists!');
+  //todo: missing 'wrong password' message in application
```
Comment (CarinaDraganJW) [inline test/account_test.js:48]: As Cameron mentioned yesterday, I think it would be nice to extract all todos in the backlog (or a separate list)

Comment (jwpradm) [reply test/account_test.js:48]: I've created a separate ticket for extracting https://jwplayer.atlassian.net/browse/BC-77

File: test/watch_history_test.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+});
+
+Scenario('I can continue watching from home immediately', async({ I })=> {
+  within('div[data-mediaid="continue-watching"]', async () => {
+    I.click('div[aria-label="Play Agent 327"]');
+    I.seeInCurrentUrl('play=1');
+  });
+});
+
+// Scenario('I can get my watch history stored to my account after login', async({ I })=> {
```
Comment (CarinaDraganJW) [inline test/watch_history_test.js:59]: Are these type of comments for future reference? 🙂

Comment (jwpradm) [reply test/watch_history_test.js:59]: Not sure, that has been done by the previous team. I suppose that's just unfinished work. I included "commented out code" in the description of the BC-77 ticket

**Review by CarinaDraganJW — APPROVED** [review]
Looks good to me 👍

File: test/watch_history_test.js (hunk 2 above)
Comment (CarinaDraganJW) [reply test/watch_history_test.js:59]: That's great 👍

File: test/account_test.js (hunk 1 above)
Comment (CarinaDraganJW) [reply test/account_test.js:48]: Great 👍

**Pull request merged**